backends/graphics/openglsdl/openglsdl-graphics.cpp

```diff
@@ -337,6 +346,29 @@ Common::List<Graphics::PixelFormat> OpenGLSdlGraphicsManager::getSupportedFormat
 #endif
 
 void OpenGLSdlGraphicsManager::updateScreen() {
+#if SDL_VERSION_ATLEAST(2, 0, 0)
+	static unsigned int lastUpdateTime = 0;
+	unsigned int currentTime;
```

Comment:
This variable can be declared inside the else-block to make it's scope minimal. (That also makes it easier to see when it's first assigned.